# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../psi/uic-issue.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur psi-0.9.3.orig/src/options/opt_appearance-ui.ui psi-0.9.3/src/options/opt_appearance-ui.ui
--- psi-0.9.3.orig/src/options/opt_appearance-ui.ui	2004-11-20 14:03:44.000000000 +0200
+++ psi-0.9.3/src/options/opt_appearance-ui.ui	2005-10-15 00:44:45.000000000 +0300
@@ -458,9 +458,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
 </includehints>
 </UI>
diff -Nur psi-0.9.3.orig/src/ui_about.ui psi-0.9.3/src/ui_about.ui
--- psi-0.9.3.orig/src/ui_about.ui	2004-04-04 12:06:27.000000000 +0300
+++ psi-0.9.3/src/ui_about.ui	2005-10-15 00:41:40.000000000 +0300
@@ -334,7 +334,6 @@
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
     <includehint>fancylabel.h</includehint>
-    <includehint>urllabel.h</includehint>
     <includehint>psitextview.h</includehint>
     <includehint>psitextview.h</includehint>
     <includehint>psitextview.h</includehint>
